lib/stdlib/test/sets_test_lib.erl

```diff
@@ -25,16 +25,21 @@
 new(Mod, Eq) ->
     new(Mod, Eq, fun Mod:new/0, fun Mod:from_list/1).
 
-new(Mod, Eq, New, FromList) ->
+new(Mod, Eq0, New, FromList) ->
+    Eq = fun(S1, S2) ->
+             IsEqual = Eq0(S1, S2),
+             IsEqual = Mod:is_equal(S1, S2)
+         end,
     fun	(add_element, {El,S}) -> add_element(Mod, El, S);
 	(del_element, {El,S}) -> del_element(Mod, El, S);
 	(empty, []) -> New();
-	(equal, {S1,S2}) -> Eq(S1, S2);
+%	(equal, {S1,S2}) -> Eq(S1, S2);
```

Comment:
Please remove this line. We prefer to have the editing history in git.

Reply:
Oops. Done.